Example/Podfile

```diff
@@ -10,3 +11,9 @@ target 'Sentinel_Example' do
     inherit! :search_paths
   end
 end
+
+target 'MacOSExample' do
```

Comment:
Update target names for both iOS and macOS - as mentioned in the previous comment.